src/elements/common/annotator-context/withAnnotations.tsx

```diff
@@ -77,6 +79,16 @@ export default function withAnnotations<P extends object>(
             annotator.emit('annotations_active_set', id);
         };
 
+        emitActivitySidebarFilterChangeEvent = (status: string) => {
+            const { annotator } = this;
+
+            if (!annotator) {
+                return;
+            }
+
+            annotator.emit('sidebar.activity_sidebar_filter_change', status);
```

Comment:
Can we move this to a callback prop on `ActivitySidebar`, instead? It's not really directly related to annotations.

Reply:
I could do that but how would we pass the `annotator` object to `ActivitySidebar`?

Reply:
Refactored as discussed.